src/poly.c

```diff
@@ -188,6 +188,8 @@ static void _span_fill(twin_pixmap_t *pixmap,
 
     /* starting address */
     s = span + (x >> TWIN_POLY_SHIFT);
+    twin_a8_t *max_pixel_ptr =
```

Comment:
Don't do this. Use `uintptr_t` instead.

Quote from Oracle Solaris 64-bit Developer's Guide:
> Using the `uintptr_t` type as the integral type for pointers is a better option than using a fundamental type such as unsigned long. Even though an unsigned long is the same size as a pointer in both the ILP32 and LP64 data models, the use of the `uintptr_t` requires only the definition of `uintptr_t` to change when a different data model is used. This makes it portable to many other systems. It is also a clearer way to express your intentions in C.
> 
> The `intptr_t` and `uintptr_t` types are extremely useful for casting pointers when you want to do address arithmetic. They should be used instead of long or unsigned long for this purpose.